apps/browser/src/safari/safari/SafariWebExtensionHandler.swift

```diff
@@ -194,6 +205,21 @@ func jsonDeserialize<T: Decodable>(json: String?) -> T? {
     }
 }
 
+extension LAContext {
+    func isBiometricsAvailable() -> Bool {
+        var error: NSError?
+
+        self.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: &error)
+
+        if let e = error, e.code != kLAErrorBiometryLockout {
+            return false;
+        }
+        else {
```

Comment:
```suggestion
        } else {
```
:pick: nit-pick: looking at the style of the rest of the file, I believe this formatting is more in line with the rest.

Reply:
Updated in https://github.com/bitwarden/clients/pull/11040/commits/9f8983a609c17ab926a551b154204a364dabc2ef